File: conformance/bundles/08-string-capture/greet.steps.rs
//! Rust sibling of `greet.steps.ts` (bundle `08-string-capture`).

use varar::{Steps, Value};

pub fn register(s: &mut Steps) {
    s.stimulus("I greet {string}", |_state, _name| Ok(None));
}

pub fn state() -> Value {
    Value::Null
}

File: conformance/bundles/08-string-capture/greet.steps.cs
// C# sibling of greet.steps.ts / .rs (bundle 08-string-capture).
using Varar;
using Varar.Core;

namespace Varar.Corpus.B08;

public static class GreetSteps
{
    public static void Register(Steps s)
    {
        s.Stimulus("I greet {string}", (state, name) => null);
    }

    public static Value State() => Value.Null;
}
